spark_auto_mapper_fhir/resources/immunization.py

```diff
@@ -1,30 +1,172 @@
 from spark_auto_mapper_fhir.resources.fhir_resource_base import FhirResourceBase
 from spark_auto_mapper_fhir.extensions.extension import Extension
 from spark_auto_mapper_fhir.complex_types.meta import Meta
+from spark_auto_mapper_fhir.resources.location import Location
+from spark_auto_mapper_fhir.resources.observation import Observation
+from spark_auto_mapper_fhir.resources.organization import Organization
+from spark_auto_mapper_fhir.resources.patient import Patient
+from spark_auto_mapper_fhir.valuesets.immunization_function import (
+    ImmunizationFunctionCode,
+)
+from spark_auto_mapper_fhir.valuesets.immunization_funding_source import (
+    ImmunizationFundingSourceCode,
+)
+from spark_auto_mapper_fhir.valuesets.immunization_origin import ImmunizationOriginCode
+from spark_auto_mapper_fhir.valuesets.immunization_program_elegibility import (
+    ImmunizationProgramElegibilityCode,
+)
+from spark_auto_mapper_fhir.valuesets.immunization_reason_code import (
+    ImmunizationReasonCode,
+)
+from spark_auto_mapper_fhir.valuesets.immunization_route import ImmunizationRouteCode
+from spark_auto_mapper_fhir.valuesets.immunization_site import ImmunizationSiteCode
+
+from spark_auto_mapper_fhir.valuesets.immunization_status_reason import (
+    ImmunizationStatusReasonCode,
+)
+from spark_auto_mapper_fhir.complex_types.identifier import Identifier
+from spark_auto_mapper_fhir.valuesets.immunization_subpotent_reason import (
+    ImmunizationSubpotentReasonCode,
+)
+from spark_auto_mapper_fhir.valuesets.immunization_target_disease import (
+    ImmunizationTargetDiseaseCode,
+)
+from spark_auto_mapper_fhir.valuesets.vaccine_code import VaccineCode
 
 
 class Immunization(FhirResourceBase):
     def __init__(
         self,
+        *,
         id_: Optional[FhirId] = None,
         meta: Optional[Meta] = None,
         extension: Optional[FhirList[Extension]] = None,
+        identifier: Optional[FhirList[Identifier]] = None,
+        status: ImmunizationSiteCode,
+        statusReason: Optional[CodeableConcept[ImmunizationStatusReasonCode]] = None,
+        vaccineCode: CodeableConcept[VaccineCode],
+        patient: Reference[Patient],
+        encounter: Optional[Reference[Encounter]] = None,
+        occurrenceDateTime: FhirDateTime,
+        occurrenceString: FhirString,
+        recorded: Optional[FhirDateTime] = None,
+        primarySource: Optional[FhirBoolean] = None,
+        reportOrigin: Optional[CodeableConcept[ImmunizationOriginCode]] = None,
+        location: Optional[Reference[Location]] = None,
+        manufacturer: Optional[Reference[Organization]] = None,
+        lotNumber: Optional[FhirString] = None,
+        expirationDate: Optional[FhirDate] = None,
+        site: Optional[CodeableConcept[ImmunizationSiteCode]] = None,
+        route: Optional[CodeableConcept[ImmunizationRouteCode]] = None,
+        doseQuantity: Optional[SimpleQuantity] = None,
+        performer: Optional[FhirList[ImmunizationFunctionCode]] = None,
```

Comment:
'performer' needs to be split out as a backbone element definition

Reply:
I created the BackboneElement but I didnt add, thank you for noticing it. I corrected it